tools/infer_simple.py

```diff
@@ -84,7 +101,7 @@ def parse_args():
         type=str
     )
     parser.add_argument(
-        'im_or_folder', help='image or folder of images', default=None
+        '--im_or_folder', help='image or folder of images', default=None
```

Comment:
Can you either update the documentation to include this flag or revert it back to a positional argument?